libraries/AP_HAL_ESP32/SdCard.cpp

```diff
@@ -254,8 +254,16 @@ void mount_sdcard()
 {
     mount_sdcard_spi();
 }
+
 #endif // end spi
 
+void unmount_sdcard()
```

Comment:
I cant see any reason why this function was moved around

Reply:
ok